webroot/src/models/User/User.model.ts

```diff
@@ -7,6 +7,12 @@
 
 import deleteUndefinedProperties from '@models/_helpers';
 
+export enum Compact {
```

Comment:
I have been taught to avoid typescript enums since they are not a strictly typed superset of javascript, and to instead use [unions](https://stackoverflow.com/questions/40275832/typescript-has-unions-so-are-enums-redundant).

That said, if this is a choice that's more consistent with this project's style conventions, I'm fine to keep as-is. Just thought I'd note.

Reply:
In this case, and in general, we're usually using enums to avoid repeated string literals in code, and to have a central place where the values are defined; enumerable if needed. They've worked well to address those higher level preferences for us.

Depending on where the backend is headed, I feel like there is a chance this enum may just evolve into a proper model. If I had some more certainty I would have done that initially, but there was a lot of churn in the v0 API (as kind of expected).

Reply:
Got it, makes sense!